m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.RPT.BATCH

```diff
@@ -0,0 +1,773 @@
+            DO [valid target transaction - add to report]
+             BNOTEXT=SEGMENT(TCCOMMENT(TCCOUNT),4,LENGTH(TCCOMMENT(TCCOUNT)))
+             CALL BNONLTS
+             BNODELIMITER=BNODELIMSPACE
+             BNODATALINEINPUT=BNOTEXT
+             CALL PARSEDATALINE
+             TOACCOUNT=BNODATALINEFIELD(BNODATAFIELDCOUNT-2)
+             TOSLIDTYPE=BNODATALINEFIELD(BNODATAFIELDCOUNT-1)
+             TOSLID=BNODATALINEFIELD(BNODATAFIELDCOUNT)
+             TOTRANAMOUNT=SHARE TRANSACTION:TRANAMOUNT
+             TRANPOSTDATE=SHARE TRANSACTION:POSTDATE
+             CALL PRINTTRANTOREPORT
+             CALL CLEARTRANVARS
+            END
+          END [IF TCCOUNT=1 THEN...]
+         ELSE IF TCCOUNT=2 THEN [transaction with tran comment]
+          DO
+           IF CHARACTERSEARCH(TCCOMMENT(TCCOUNT-1),TARGETCOMMENT)>0 THEN
+            DO
+             BNOTEXT=SEGMENT(TCCOMMENT(TCCOUNT),4,LENGTH(TCCOMMENT(TCCOUNT)))
+             CALL BNONLTS
+             BNODELIMITER=BNODELIMSPACE
+             BNODATALINEINPUT=BNOTEXT
+             CALL PARSEDATALINE
+
+             TOACCOUNT=BNODATALINEFIELD(BNODATAFIELDCOUNT-2)
+             TOSLIDTYPE=BNODATALINEFIELD(BNODATAFIELDCOUNT-1)
+             TOSLID=BNODATALINEFIELD(BNODATAFIELDCOUNT)
+             TOTRANAMOUNT=SHARE TRANSACTION:TRANAMOUNT
+             TRANPOSTDATE=SHARE TRANSACTION:POSTDATE
+             CALL PRINTTRANTOREPORT
+             CALL CLEARTRANVARS
+            END
+          END [IF TCCOUNT=2]
+        END [IF SHARE TRANSACTION:ACTIONCODE<>"W"]
+      END [SHARE TRANSACTION:COMMENTCODE<>1]
+    END [FOR EACH SHARE TRANSACTION WITH...]
+   UNTIL SHARE TRANSACTION:POSTDATE<TARGETSTARTDATE 
+  END [FOR EACH SHARE]
+
+ CALL CLEARTRANVARS
+ FOR EACH LOAN
+  DO
+   TRANSOURCE=TRANSOURCELOAN
+   TRANSOURCEID=LOAN:ID
+   FOR EACH LOAN TRANSACTION WITH (LOAN TRANSACTION:ACCOUNTNUMBER=ACCOUNT:NUMBER AND
+                                    LOAN TRANSACTION:POSTDATE>=TARGETSTARTDATE AND
+                                   (LOAN TRANSACTION:POSTDATE<=TARGETENDDATE OR
+                                    TARGETENDDATE=DATENULL) AND
+                                    LOAN TRANSACTION:VOIDCODE=0 AND
+                                    LOAN TRANSACTION:USERNUMBER=TARGETSYMXUSER)
+    DO
+     IF LOAN TRANSACTION:COMMENTCODE=1 THEN
+      DO
+       IF TCCOUNT<TCCOUNTMAX THEN [store pertinant data for future use]
+        DO
+         TCCOUNT=TCCOUNT+1
+         TCCOMMENT(TCCOUNT)=LOAN TRANSACTION:COMMENT
+         TCIDTYPE(TCCOUNT)=LOAN TRANSACTION:IDTYPE
+         TCID(TCCOUNT)=LOAN TRANSACTION:ID
+         TCSTMTDESCR(TCCOUNT)=LOAN TRANSACTION:STMTDESC:1
+        END
+      END
+     ELSE [transaction is not a comment - validate & process]
+      DO
+       IF LOAN TRANSACTION:ACTIONCODE<>"A" THEN
+        CALL CLEARTRANVARS
+       ELSE
+        DO
+         IF TCCOUNT=1 THEN [transaction with statement description]
```

Comment:
See above comments in SHARE section TCCOMMENT references.

Reply:
@rrobison44 done.